client/lib/keyboard.js

```diff
@@ -36,6 +44,15 @@ Mousetrap.bind(['down', 'up'], (evt, key) => {
   }
 });
 
+Mousetrap.bind('space', () => {
+  if (!Session.get('currentCard')) {
+    return;
+  }
+  const currentUserId = Meteor.userId();
+  const card = Cards.findOne(Session.get('currentCard'));
+  card.toggleMember(currentUserId);
```

Comment:
I think you should explicitly handle the case when `currentUserId` is `null`, and verifying that the user is a board member (`Meteor.user().isBoardMember()`) before executing the action